joomla/joomla-cms
# [SQL errors] When installing languages (backend/installation) in PDO Strict mode/Sql Azure

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/9457 and new issue (in MSSQL).
### Summary of Changes

This solves languages (and probably other extensions) install problems in MSSQL and "PDO (MySQL)" in strict mode.
### Testing Instructions
- For "PDO (MySQL)" see https://github.com/joomla/joomla-cms/issues/9457
-  For "MSSQL" try to install languages on install or after install before patch (don't work), after patch (should work).
- Test also upgrade process.
- Code review.
### Documentation Changes Required

None.

@alikon @waader since you are the db experts please check and review.
